jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -403,93 +405,92 @@ export class SortingView extends DOMWidgetView {
   }
 }
 
-export class RecordingViewModel extends DOMWidgetModel {
-  initialize(attributes: any, options: any) {
-    super.initialize(attributes, options);
-  }
-
-  defaults() {
-    return {
-      ...super.defaults(),
-      _model_name: RecordingViewModel.model_name,
-      _model_module: RecordingViewModel.model_module,
-      _model_module_version: RecordingViewModel.model_module_version,
-      _view_name: RecordingViewModel.view_name,
-      _view_module: RecordingViewModel.view_module,
-      _view_module_version: RecordingViewModel.view_module_version,
-      pluginName: '',
-      recordingObject: {}
-    };
-  }
-
-  static serializers: ISerializers = {
-    ...DOMWidgetModel.serializers,
-    // Add any extra serializers here
-  };
-
-  static model_name = 'RecordingViewModel';
-  static model_module = MODULE_NAME;
-  static model_module_version = MODULE_VERSION;
-  static view_name = 'RecordingView'; // Set to null if no view
-  static view_module = MODULE_NAME; // Set to null if no view
-  static view_module_version = MODULE_VERSION;
-}
-
-export class RecordingView extends DOMWidgetView {
-  _hitherJobManager: HitherJobManager
-  initialize() {
-    this._hitherJobManager = new HitherJobManager(this.model)
-  }
-  element() {
-    const pluginName = this.model.get('pluginName')
-    const recordingObject = this.model.get('recordingObject')
-    const recordingInfo = this.model.get('recordingInfo')
-    const plugin = extensionContext._recordingViewPlugins[pluginName]
-    if (!plugin) return <div>Plugin not found: {pluginName}</div>
-
-    const example = exampleSorting()
-
-    const hitherContext: HitherContext = {
-      createHitherJob: (functionName: string, kwargs: {[key: string]: any}, opts: HitherJobOpts): HitherJob => {
-        return this._hitherJobManager.createHitherJob(functionName, kwargs, opts)
-      }
-    }
-
-    const recording = {
-      recordingId: '',
-      recordingLabel: '',
-      recordingObject,
-      recordingPath: '',
-      recordingInfo
-    }
-
-    const plugins: Plugins = {
-      recordingViews: extensionContext._recordingViewPlugins,
-      sortingViews: extensionContext._sortingViewPlugins,
-      sortingUnitViews: extensionContext._sortingUnitViewPlugins,
-      sortingUnitMetrics: extensionContext._sortingUnitMetricPlugins
-    }
-
-    // this.el.textContent = this.model.get('value') + ' --- test9';
-    const x = (
-      <plugin.component
-        recording={recording}
-        plugins={plugins}
-        hither={hitherContext}
-        calculationPool={calculationPool}
-        recordingSelection={{}}
-        recordingSelectionDispatch={(a: RecordingSelectionAction) => {}}
-      />
-    )
-    return x
-  }
-  render() {
-    // this.el.classList.add('custom-widget');
+// export class RecordingViewModel extends DOMWidgetModel {
```

Comment:
Commenting out the recording views for now. Will focus on sorting view sfor now, and will restore the recording views later.